tailwind.config.js

```diff
@@ -4,6 +4,14 @@ module.exports = {
   },
   darkMode: false, // or 'media' or 'class'
   theme: {
+    fontSize: {
+      xs: ".75rem",
+      sm: ".875rem",
+      default: ".875rem",
+      lg: "1.125rem",
+      xl: "1.25rem",
+      "2xl": "1.5rem",
+    },
```

Comment:
We don't need to hardcore all the other font sizes except for `default` here, do we?

Reply:
We don't strictly need to yet, but this is setting up follow-up work I'm in the process of. Would like to keep it for now, and we can always strip it out later when I'm done.